components/automate-opensearch/habitat/hooks/post-run

```diff
@@ -18,35 +23,35 @@ HOST="127.0.0.1"
 ~}}
 
 {{#if cfg.disable ~}}
-# ElasticSearch has been disabled via config, exit early.
+# OpenSearch has been disabled via config, exit early.
 touch ${POST_RUN_SENTINEL}
 exit 0
 
 {{/if ~}}
 
-{{#if cfg.deprecated.external_es ~}}
-# We're using the deprecated external ElasticSearch proxy node, exit early.
+{{#if cfg.deprecated.external_os ~}}
+# We're using the deprecated external OpenSearch proxy node, exit early.
```

Comment:
what is a deprecated external open search proxy node?

Reply:
this flag was used in elastic sidecar was getting waring after removing this.